atlasdb-feign/src/main/java/com/palantir/atlasdb/http/AtlasDbFeignTargetFactory.java

```diff
@@ -49,7 +51,9 @@
             10_000, 65_000);
 
     private static final ObjectMapper mapper = new ObjectMapper()
-            .registerModule(new Jdk8Module());
+            .registerModule(new Jdk8Module())
+            .registerModule(new JavaTimeModule())
+            .registerModule(new GuavaModule());
```

Comment:
JavaTimeModule is required for `Duration` in `Lease`, and GuavaModule is needed to deserialize immutableSet in `LeasableRefreshLockResponse`